ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template5.kt

```diff
@@ -451,8 +511,8 @@ private fun Template5PaywallPreview() {
 }
 
 @OptIn(ExperimentalPreviewRevenueCatUIPurchasesAPI::class)
-@Preview(showBackground = true, locale = "en-rUS")
-@Preview(showBackground = true, locale = "es-rES")
+@Preview(showBackground = true, locale = "en-rUS", group = "footer")
```

Comment:
This allows showing only a subset of previews which is super useful.

Reply:
Nice! We definitely had a ton of previews, this will help!